File: SQLReader/Documentation/SQLReader/SourceCode/Enumeration/cs/SQLReader.File_Type.cs
public enum File_Type : byte {
    
    // Hex value is: 00000000
    [Description("Rows")]
    ROWS = 0,
    
    // Hex value is: 00000001
    [Description("Log")]
    LOG = 1,
    
    // Hex value is: 00000004
    [Description("Full-text")]
    FULLTEXT = 4,
}

File: SQLReader/Documentation/SQLReader/SourceCode/Enumeration/vb/SQLReader.File_Type.vb
Public Enum File_Type As Byte
    
    'Hex value is: 00000000
    <Description("Rows")>  _
    ROWS = 0
    
    'Hex value is: 00000001
    <Description("Log")>  _
    LOG = 1
    
    'Hex value is: 00000004
    <Description("Full-text")>  _
    FULLTEXT = 4
End Enum
